README.md

```diff
@@ -81,6 +81,15 @@ skpm link path-to-local-plugin
 
 It will also ask you if you want to [disable the caching mechanism](http://developer.sketchapp.com/introduction/preferences/#always-reload-scripts-before-running) and force Sketch to always reload a Plugin’s code from disk (recommended when developing).
 
+
+#### Watching for changes
+
+To automatically rebuild and link your plugin as you make changes:
```

Comment:
so you don't need to `link` the plugin to watch it. Once you have linked the plugin, any changes will be reflected directly on sketch (if you disable the caching mechanism, cf. l.82).
The `watch` option is just here to look at the sources and compile them when there is a change. As it is not another command, I would put it under the build command